src/store/user-store.js

```diff
@@ -1,16 +1,48 @@
-import { makeAutoObservable } from "mobx";
+import { makeAutoObservable, flow } from "mobx";
 
 class UserStore {
   isLoggedIn = false;
 
+  userResponse = {};
+
   constructor(rootStore) {
     this.rootStore = rootStore;
-    makeAutoObservable(this);
+    makeAutoObservable(this, {
+      fetchUserInformation: flow
+    });
```

Comment:
Step 3: In our `makeAutoObservable` function inside of the class constructor, we're setting `fetchUserInformation: flow`.

Don't get too worried about the details here. Just know that this is `mobx` way of creating asynchronous functions inside the stores.